.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: enabled
+contact_links:
+  - name: Cryostat Community Support
+    url: https://github.com/cryostatio/cryostat/discussions
```

Comment:
Should this be `https://github.com/cryostatio/cryostat3/discussions` that points to this repo?